src/components/UriViewer/LogViewer.ts

```diff
@@ -0,0 +1,231 @@
+    setActiveTextContent({
+      status: 'Loading',
+      state: undefined,
+    });
+    setActiveDownloadUri(undefined);
+
+    // tab switching set the currently active log (which triggers this effect). Fetch the content for the new log.
+    const uri = currentlyActiveLog?.logUri;
+    if (!_.isEmpty(uri)) {
+      fetch(uri);
+    }
+  }, [logs, currentlyActiveLog, fetchLogContent]);
+
+  const renderActiveTextContent = () => {
+    switch (activeTextContent.status) {
+      case 'Loading':
+        return div([centeredSpinner()]);
+      case 'Error':
+        return div([activeTextContent.error.message]);
+      case 'Ready':
+        return div([activeTextContent.state]);
+      default:
+        return div(['Unknown error']);
+    }
+  };
+
+  const tabsArray: SimpleTabProps[] = logs.map((log) => {
+    return { key: log.logKey, title: log.logTitle, width: tabMaxWidth };
+  });
+
+  return h(
+    Modal,
+    {
+      onDismiss,
+      title: modalTitle,
+      titleChildren: h(
+        InfoBox,
+        {
+          style: { marginLeft: '1ch' },
+          tooltip: undefined,
+          size: undefined,
+          side: undefined,
+          iconOverride: undefined,
+        },
+        [
+          dl([
+            dt({ style: { fontWeight: 'bold' } }, ['Execution:']),
+            dd({ style: { marginBottom: '0.5rem' } }, [
+              'Each workflow has a single execution log which comes from the engine running your workflow. Errors in this log might indicate a Terra systems issue, or a problem parsing your WDL.',
+            ]),
+            dt({ style: { fontWeight: 'bold' } }, ['Task Standard Out/Error:']),
+            dd({ style: { marginBottom: '0.5rem' } }, [
+              "Task logs are from user-defined commands in your WDL. You might see an error in these logs if there was a logic or syntax error in a command, or if something went with the tool you're running.",
+            ]),
+            dt({ style: { fontWeight: 'bold' } }, ['Backend Standard Out/Error:']),
+            dd({ style: { marginBottom: '0.5rem' } }, [
+              "Backend logs are from the Azure Cloud compute job that prepares your task to run and cleans up after. You might see errors in these logs if the there was a problem downloading the task's input files or pulling its container, or if something went wrong on the compute node while the task was running.",
+            ]),
+          ]),
+        ]
+      ),
+      showCancel: false,
+      showX: true,
+      showButtons: false,
+      width: modalMaxWidth,
+    },
+    [
+      h(
+        SimpleTabBar,
+        {
+          value: currentlyActiveLog?.logKey,
+          'aria-label': 'Log file tabs',
+          onChange: (key: string) => {
+            const newLog = logs.find((log) => log.logKey === key);
+            setCurrentlyActiveLog(newLog);
+          },
+          tabs: tabsArray,
+        },
+        []
```

Comment:
SimpleTabBar renders its children wrapped in a `tabpanel`. Thus, the active tab's content must be rendered here in order for the elements to be properly linked and the tabs to be accessible.